### ISSUE TITLE
Integration with Thunderbird 68.4.1

### ISSUE THREAD

**Issue by alix-guillard-deltatre** [opto/nostalgy#3]
Hi,
it looks like the shortcuts with the status bar are again not working after the latest Thunderbird update to 68.4.1. Anyone confirmed?

**Comment by hubenduben** [comment]
... same here. Pressing "s" marked the message. :-/  

Using "Preferences" / "Keys" solved it: Save Message was displayed as "s" (did not work), changed it do "x" (displayed as "X", worked!), changed it back to "s" (now displayed as "S", working) 

Win10 1909 (clean install), Thunderbird 68.4.2 (64Bit), Nostalgy 1.0.21.

**Comment by alix-guillard-deltatre** [comment]
I don't have "Preferences" / "Keys" on Mac. do you use a specific add on to get this menu?

**Comment by hubenduben** [comment]
Yes, this is a strange behavior: TB60 uses "the new" Add-On-Pref Menu (3rd line, Add-On Prefences), in TB68 any of the Add-On use this Menu. Prefs of Add-On are found somewhere in the "Tool Menu" (here: 5th line: "Manualy Sort Foler", 3rd last "Remove Duplicate Messages", last "nostalgy").  

![Screenshot 2020-01-28 at 13 10 45](https://user-images.githubusercontent.com/60383162/73263441-dc1b4880-41d0-11ea-9fab-27a47fc3a655.png)

**Comment by alix-guillard-deltatre** [comment]
Found it! Cheers.

**Comment by philgp** [comment]
> Found it! Cheers.

Where did you find it? My Thunderbird just updated to 78.2.2 and I have this Nostalgy installed (1.0.25), but can no longer find its menu. @alixguillard

**Comment by alix-guillard-deltatre** [comment]
@philgp I found it in Tools > Nostalgy… or Menu > Add-ons > Add-on Preferences > Nostalgy
(Thunderbird 68.12.0 Nostalgy 1.0.21)

**Comment by philgp** [comment]
@alixguillard Odd. On my MacOS Thunderbird 78.2.2: No Nostalgy under Tools (I think that's OK, it's the old method). But also not under Tools -> Add-on Options -> (I have one add-on listed there, but it's not Nostalgy).

**Comment by asolochek** [comment]
I just updated to thunderbird 78.4.0 and I don't see the nostalgy options, but I believe that is because thunderbird disabled the add-on saying it's not compatible. How is compatibility determined?

**Comment by aloxe** [comment]
@asolochek If the add-on uses a library that has been changed with the Thunderbird update, it may break so it is flagged as not compatible.
